src/internal/metrics/metrics.proto

```diff
@@ -22,41 +22,40 @@ message Metrics {
-    int64 cfg_standby              = 19; // Number of pipelines with Standby congigured
-    int64 cfg_s3gateway            = 20; // Number of pipelines with S3 Gateway configured
-    int64 cfg_services             = 21; // Number of pipelines with services configured
-    int64 cfg_errcmd               = 22; // Number of pipelines with error cmd set
-    int64 cfg_stats                = 23; // Number of pipelines with stats configured
-    int64 cfg_tfjob                = 24; // Number of pipelines with TFJobs configured
-    int64 input_group              = 25; // Number of pipelines with group inputs
-    int64 input_join               = 26; // Number of pipelines with join inputs
-    int64 input_cross              = 27; // Number of pipelines with cross inputs
-    int64 input_union              = 28; // Number of pipelines with union inputs
-    int64 input_cron               = 29; // Number of pipelines with cron inputs
-    int64 input_git                = 30; // Number of pipelines with git inputs
-    int64 input_pfs                = 31; // Number of pfs inputs
-    int64 input_commit             = 32; // Number of pfs inputs with commits
-    int64 input_join_on            = 33; // Number of pfs inputs with join_on
-    int64 input_outer_join         = 34; // Number of pipelines with outer joins
-    int64 input_lazy               = 35; // Number of pipelines with lazy set
-    int64 input_empty_files        = 36; // Number of pipelines with empty files set
-    int64 input_s3                 = 37; // Number of pipelines with S3 input
-    int64 input_trigger            = 38; // Number of pipelines with triggers set
-    float resource_cpu_req         = 39; // Total CPU request across all pipelines
-    float resource_cpu_req_max     = 40; // Max CPU resource requests set
-    string resource_mem_req        = 41; // Sting of memory requests set across all pipelines
-    int64 resource_gpu_req         = 42; // Total GPU requests across all pipelines
-    int64 resource_gpu_req_max     = 43; // Max GPU request across all pipelines
-    string resource_disk_req       = 44; // String of disk requests set across all pipelines
-    float resource_cpu_limit       = 45; // Total CPU limits set across all pipelines
-    float resource_cpu_limit_max   = 46; // Max CPU limit set
-    string resource_mem_limit      = 47; // String of memory limits set
-    int64 resource_gpu_limit       = 48; // Number of pipelines with
-    int64 resource_gpu_limit_max   = 49; // Max GPU limit set
-    string resource_disk_limit     = 50; // String of disk limits set across all pipelines
-    uint64 max_parallelism         = 51; // Max parallelism set
-    uint64 min_parallelism         = 52; // Min parallelism set
-    uint64 num_parallelism         = 53; // Number of pipelines with parallelism set
+    int64 cfg_egress               = 17; // Number of pipelines with Egress configured
```

Comment:
I would not change the number. Just mark the field unused. An old interpreter will read "17" as data for pps_build.

Reply:
Hmm, I thought the plan was to make protos explicitly incompatible between 1.x and 2.x, but I can preserve the numbers